etna/records/models.py

```diff
@@ -53,6 +53,7 @@ class Record(DataLayerMixin, APIModel):
     template_summary_title: str = ""
     hierarchy_level3_reference_number: str = ""
     hierarchy_level3_summary_title: str = ""
+    related_materials: dict = field(default_factory=dict)
```

Comment:
`related_materials` is a `list` of `dicts`, isn't it?

Reply:
Not sure at this time why `dict` still works, but setting it to list for readability.